### ISSUE TITLE
[Question] Whole suite passed but single file failed?

### ISSUE THREAD

**Issue by hoangngocanhskedulo** [microsoft/playwright#20177]
Hi,

Has anyone observed this issue before when each single file of a whole test suite is executed and passed but when the suite itself is executed at once and some are failed randomly? I've seen this way too many times and have no idea how to fix.

I really appreciate if anyone has any ideas or solutions.

Thanks,

**Comment by yury-s** [comment]
There can be many reasons why the tests would fail while running in parallel, e.g. concurrent access to some resource as playwright runs the tests in parallel. We need a concrete example where you observe the problem and that we could run locally to make this report actionable. Please file a new issue with a repro and link it to this one.
